tests/storage/cdi_clone/conftest.py

```diff
@@ -31,6 +31,32 @@ def cirros_dv_with_block_volume_mode(
     )
 
```

Comment:
tox is failing in CI, most likely because `cirros_dv_with_filesystem_volume_mode` and `cirros_dv_with_block_volume_mode` fixtures are not used anymore, which means you can delete them